client/app/bundles/shared/components/Checkbox.jsx

```diff
@@ -0,0 +1,45 @@
+// @flow
+import React from 'react';
+import css from './Checkbox.scss';
+
+type Props = {
+  id?: string;
+};
+
+type State = {
+  checkboxState: boolean;
```

Comment:
I would rename this to checked so it's more explicit what this state is referring to